docs/content/dagster-cloud/getting-started.mdx

```diff
@@ -341,11 +341,12 @@ To set up continuous integration using GitHub Actions, you can use your own Dags
 1. In the repository, set the necessary GitHub action secrets:
 
    - `DAGSTER_CLOUD_API_TOKEN` - The Dagster Cloud agent token from the previous section. Refer to the [agent tokens documentation](/dagster-cloud/account/managing-user-agent-tokens#managing-agent-tokens) for more info.
-   - `ORGANIZATION_ID` - The organization ID of your Dagster Cloud organization, found in your account's URL. For example, if your organization is found at `https://dagster.cloud/pied-piper` or `https://pied-piper.dagster.cloud/`, this would be `pied-piper`.
 
    Refer to the [GitHub docs](https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository) for more info about GitHub Action secrets.
 
-2. In both workflow files, uncomment the step that is relevant to your Docker container registry. For example, if using DockerHub, you'd uncomment the `DockerHub` step in these files.
+2. In `deploy-dagster-cloud.yml`, set the `DAGSTER_CLOUD_ORGANIZATION` environment variable to your dagster.cloud organization name.
```

Comment:
I agree. I think all of this info in the "getting started" page is a bit overwhelming. Many of the links point to later sections in the docs because you need to know about code locations and tokens before you can implement CI/CD.

I'm now thinking we should have a separate page dedicated to setting up CI/CD somewhere under the Cloud section: 
<img width="277" alt="image" src="https://github.com/dagster-io/dagster/assets/7066873/f0ed7a59-521b-4c86-913b-6b849f884aa2">

Reply:
We should. Branch Deployments currently serves the CI portion of this in the docs, but it's not immediately clear that that's what that term means.

Should CI/CD go under **Managing deployments**?

Reply:
For now I've added a link to our quickstart's workflow file and also added some info about what happens when the workflow runs.